cocos/ui/UIWidget.cpp

```diff
@@ -558,6 +558,10 @@ void Widget::setTouchEnabled(bool enable)
     if (_touchEnabled)
     {
         _touchListener = EventListenerTouchOneByOne::create();
+        if (!_touchListener) {
```

Comment:
From the code for create:
```
EventListenerTouchOneByOne* EventListenerTouchOneByOne::create()
{
    auto ret = new (std::nothrow) EventListenerTouchOneByOne();
    if (ret && ret->init())
    {
        ret->autorelease();
    }
    else
    {
        CC_SAFE_DELETE(ret);
    }
    return ret;
}
```
If I understand this correctly, then the method could return a null if we do not have enough memory to create an EventListener, which I believe an assert should be in place for just in case.